pkgs/development/python-modules/fasm/default.nix

```
@@ -0,0 +1,18 @@
+{ lib, buildPythonPackage, fetchPypi }:
```

One entry per line will make the diff easier to read in the future.

```suggestion
{ lib
, buildPythonPackage
, fetchPypi
}:
```